huntflow_api_client/entities/dictionaries.py

```diff
@@ -0,0 +1,49 @@
+from huntflow_api_client.entities.base import (
+    BaseEntity,
+    CreateEntityMixin,
+    GetEntityMixin,
+    ListEntityMixin,
+    UpdateEntityMixin,
+)
+from huntflow_api_client.models.request.dictionaries import (
+    DictionaryCreateRequest,
+    DictionaryUpdateRequest,
+)
+from huntflow_api_client.models.response.dictionaries import (
+    DictionariesListResponse,
+    DictionaryCreateResponse,
+    DictionaryResponse,
+    DictionaryUpdateResponse,
+)
+
+
+class Dictionary(BaseEntity, UpdateEntityMixin, ListEntityMixin, CreateEntityMixin, GetEntityMixin):
```

Comment:
`class Dictionary(BaseEntity, CRUDEntityMixin)`

Reply:
CRUDEntityMixin has DeleteEntityMixin. Instead, we need an UpdateEntityMixin.